clients/common/norm.cpp

```diff
@@ -66,16 +66,15 @@ double norm_check_general<rocblas_half>(char norm_type,
                                         rocblas_half* hCPU,
                                         rocblas_half* hGPU)
 {
-    // norm type can be O', 'I', 'F', 'o', 'i', 'f' for one, infinity or Frobenius norm
+    // norm type can be 'O', 'I', 'F', 'o', 'i', 'f' for one, infinity or Frobenius norm
     // one norm is max column sum
     // infinity norm is max row sum
     // Frobenius is l2 norm of matrix entries
 
     double error_double = std::numeric_limits<double>::quiet_NaN();
 
-    std::unique_ptr<float[]> hCPU_float(new float[N * lda]());
-    std::unique_ptr<float[]> hGPU_float(new float[N * lda]());
-    for(int i = 0; i < N * lda; i++)
+    host_vector<float> hCPU_float(N * lda), hGPU_float(N * lda);
+    for(rocblas_int i = 0; i < N * lda; i++)
     {
```

Comment:
`rocblas_int` should be used instead of `int` in case `rocblas_int` is larger than `int`.